beps/pub-sub.rst

```diff
@@ -29,6 +29,8 @@ The pub/sub protocol is based on the DHT protocol [#BEP-5]_ with the following m
 
 The value of K is set to one.
 
+The routing table's home bucket and its sibling SHOULD have a capacity of at least 8 nodes each. When the home bucket is split all but one node from the former home bucket's sibling SHOULD be evicted from the routing table.
```

Comment:
nevermind, i misread that.

It might be better to rephrase that as the previous sibling (now uncle) bucket should be resized to 𝒌=1